annotator-core/src/test/java/edu/ucr/cs/riple/core/CoreTest.java

```diff
@@ -554,4 +555,62 @@ public void impactedLambdaAndMemberReferenceReturnNullableTest() {
         .toDepth(1)
         .start();
   }
+
+  @Test
+  public void fieldNoInitialization() {
+    coreTestHelper
+        .addInputLines(
+            "A.java",
+            "package test;",
+            "import java.util.Objects;",
+            "public class A {",
+            "   Object f;",
+            "   A() { }",
+            "   void run() {",
+            "       this.f = foo();",
+            "   }",
+            "   Object foo() {",
+            "        return null;",
+            "   }",
+            "}")
+        .toDepth(5)
+        .disableBailOut()
+        .addExpectedReports(
+            new TReport(
+                new OnMethod("A.java", "test.A", "foo()"),
+                -2,
+                Set.of(new OnField("A.java", "test.A", Collections.singleton("f"))),
+                Collections.emptySet()),
+            new TReport(new OnField("A.java", "test.A", Collections.singleton("f")), -1))
```

Comment:
This one is due to missing initialization, correct?

Reply:
Yes, but I just remembered that this test will fail for configuration caching disabled. I moved it to a separate PR which includes the fix as well. Please see #148.